dali/test/python/checkpointing/test_dali_stateless_operators.py

```diff
@@ -162,6 +233,203 @@ def test_transforms_translation_stateless():
     check_no_input(fn.transforms.translation, 'cpu', offset=(4, 3))
 
 
+@params('cpu', 'gpu')
+def test_one_hot_stateless(device):
+    check_single_input(fn.one_hot, device)
+
+
+def test_median_bluer_stateless():
+    check_single_input(fn.experimental.median_blur, 'gpu')
+
+
+@params('cpu', 'gpu')
+def test_erase_stateless(device):
+    check_single_input(fn.erase, device, anchor=(3, 4), shape=(5, 6))
+
+
+@params('cpu', 'gpu')
+def test_pad_stateless(device):
+    check_single_input(fn.pad, device, shape=(100, 100, 3))
+
+
+@params('cpu', 'gpu')
+def test_constant_stateless(device):
+    check_no_input(fn.constant, device, idata=[1, 2, 3])
+
+
+@params('cpu', 'gpu')
+def test_reshape_stateless(device):
+    check_single_input(fn.reshape, device, shape=[1, -1])
+
+
+@params('cpu', 'gpu')
+def test_lookup_table_stateless(device):
+    check_single_input(fn.lookup_table, device, keys=[0], values=[1], default_value=123)
+
+
+@params('cpu', 'gpu')
+def test_transpose_stateless(device):
+    check_single_input(fn.transpose, device, perm=[2, 0, 1])
+
+
+def test_paste_stateless():
+    check_single_input(fn.paste, 'gpu', fill_value=0, ratio=2)
+
+
+@params('cpu', 'gpu')
+def test_color_space_conversion_stateless(device):
+    check_single_input(fn.color_space_conversion, device,
+                       image_type=dali.types.DALIImageType.RGB,
+                       output_type=dali.types.DALIImageType.YCbCr)
+
+
+def test_resize_crop_mirror_stateless(device):
+    check_single_input(fn.resize_crop_mirror, 'cpu', crop=(2, 2, 3), mirror=True)
+
+
+@params('cpu', 'gpu')
+def test_slice_stateless(device):
+    check_single_input(fn.slice, device, rel_start=(0.25, 0.25), rel_end=(0.75, 0.75))
+
+
+@params('cpu', 'gpu')
+def test_shapes_stateless(device):
+    check_single_input(fn.shapes, device)
+
+
+@params('cpu', 'gpu')
+def test_per_frame_stateless(device):
+    check_single_input(fn.per_frame, device, replace=True)
+
+
+@params('cpu', 'gpu')
+def test_get_property_stateless(device):
+    check_single_input(fn.get_property, device, key='layout')
+
+
+@params('cpu', 'gpu')
+def test_jpeg_compression_distortion_stateless(device):
+    check_single_input(fn.jpeg_compression_distortion, device)
+
+
+@params('cpu', 'gpu')
+def test_multi_paste_stateless(device):
+    check_single_input(fn.multi_paste, device, in_ids=list(range(batch_size)),
+                       output_size=[100, 100])
+
+
+@params('cpu', 'gpu')
+def test_grid_mask_stateless(device):
+    check_single_input(fn.grid_mask, device)
+
+
+@params('cpu', 'gpu')
+def test_preemphasis_filter_stateless(device):
+    check_single_input(fn.preemphasis_filter, device)
+
+
+def test_optical_flow_stateless():
+    check_single_sequence_input(fn.optical_flow, 'gpu')
```

Comment:
You may need to add (as it is done in `test_optical_flow` in `test_dali_variable_batch_size.py`, because the optical flow needs hardware support and some setups in CI do provide it. So depending on the luck, the CI may or may not pass.

```
if not is_of_supported():
        raise nose.SkipTest('Optical Flow is not supported on this platform')
```

Reply:
Oh, thank you!